### PR TITLE
btn #26 functionality

### PR DESCRIPTION
**Pull request by nicolaskolbenschlag**
Signed-off-by: nicolaskolbenschlag <nicolas.kolbenschlag@googlemail.com>

### CODE REVIEW

File: Apps/frontend/src/views/Oscilloscope.svelte [hunk 1]
```diff
@@ -18,7 +21,7 @@
 
     socket.onmessage = (message) => {
       let samples = new Float64Array(message.data);
-
+      if (!isEnabled) { return; }
```
Comment (leandertolksdorf) [inline Apps/frontend/src/views/Oscilloscope.svelte:24]: braces aren't needed here but that's a style preference.

File: Apps/frontend/src/views/Oscilloscope.svelte [hunk 2]
```diff
@@ -31,6 +34,7 @@
 <div class="wrapper" data-cy="oscilloscope">
   <div class="stack coordinate-system">
     <CoordinateSystem yScale={scaleY} />
+    <OnOffButton on:switch-plot-enabled={(e) => {isEnabled = e.detail.enabled;}} />
```
Comment (leandertolksdorf) [inline Apps/frontend/src/views/Oscilloscope.svelte:37]: I'd prefer the OnOffButton outside of the div.wrapper to 1. prevent weird any absolute positioning bugs
2. have the controls separated from the representation

**Comment by leandertolksdorf** [comment]
wrote some comments and while you're on it, could you maybe rename SineWave.svelte to something more general, e.g. Wave or Plot?

**Review by leandertolksdorf — CHANGES_REQUESTED** [review]
.

**Comment by leandertolksdorf** [comment]
Tests are missing.

**Comment by motschel123** [comment]
I am not sure about this in combination with the new #69 , what button are we using?

**Comment by nicolaskolbenschlag** [comment]
Probably we will delete this PR as the tickets #26 and #28 are duplicates.

**Pull request merged**